changelogs/current.yaml

```
@@ -152,6 +152,10 @@ bug_fixes:
   change: |
     When performing a token refresh and forwarding tokens upstream, replace existing token cookies rather than appending as
     another Cookie header.
+- area: xds
+  change: |
+    Fixed dumpEcdsFilterConfigs when not have any ecds configs.
```

```suggestion
    Removing the ECDS config entry from the config dump when it does not exist (or has expired).
```

Please double check that what I wrote reflects the actual change.